pepper-apis/studybuilder-cli/studies/pancan/patches/pedhcc-new-gastro-cancers.conf

```diff
@@ -0,0 +1,49 @@
+{
+  "questions": [
+    "DIAGNOSIS_TYPE",
+    "PRIMARY_CANCER_SELF",
+    "PRIMARY_CANCER_CHILD",
+    "PRIMARY_CANCER_ADD_CHILD",
+  ],
+
+  "groupSid": "C_GROUP_GASTRO_GASTROINTESTINAL_CANCERS",
+
+  "options": [
+    {
+      "stableId": "C_GASTRO_LIVER_HEPATOCELULAR_CANCER",
```

Comment:
Typos (in more than 1 places): It should be two `L` in word `Hepatocellular`.

Reply:
fixed.